Samples/WindowsCopilotRuntime/cs-wpf-sparse/README.md

```diff
@@ -0,0 +1,57 @@
+# Sparse Package with WPF
+
+A sparse package allows unpackaged win32 apps to gain package identity without fully
+adopting the MSIX packaging format. This is particularly useful for applications that are not yet
+ready to have all their content inside an MSIX package but still need to use Windows extensibility
+features that require package identity.
+
+# cs-wpf-sparse Sample
+
+This folder contains the cs-wpf-sparse sample which uses sparse package built on top of the WPF
+platform using WCR. BuildSparsePackage.ps1 is provided for the user which closely follows the steps
+from the link below
+
+https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/grant-identity-to-nonpackaged-apps#add-the-package-identity-metadata-to-your-desktop-application-manifest
+
+BuildSparsePackage.ps1 does the following:
+1) Build the solution in the desired platform and configuration provided in the parameters
+ 
+2) Run MakeAppx with the /nv option on the folder containing the AppxManifest
+    (cs-wpf-sparse\WCRforWPF\AppxManifest.xml) 
+    
+    - The /nv flag is required to bypass validation of referenced file paths in the manifest. 
+    
+    - The output folder is set to the output of the binaries when we build the solution regularly
+    (bin\$Platform\$Configuration\net8.0-windows10.0.22621.0) so that the MSIX will be side by side
+    to the executable built in step 1. This placement is necessary because the exe expects the external location of
+    the MSIX to be the same as it's file location.
+
+3) Run SignTool to sign the MSIX
+
+# Initialization code
+
+Just doing the steps above does not yet grant the app package identity. The MSIX package still needs to
+be registered with the external location. 
+
+The registerSparsePackage() function in cs-wpf-sparse\WCRforWPF\App.xaml.cs does this for us. This
```

Comment:
nit: need capital 'R' at the start of `registerSparsePackage()` to match the code

Reply:
(and again below)